pkg/autoops/api/api.go

```diff
@@ -879,19 +949,55 @@ func (s *AutoOpsService) isNoUpdateAutoOpsRuleCommand(req *autoopsproto.UpdateAu
 
 func (s *AutoOpsService) createUpdateAutoOpsRuleCommands(req *autoopsproto.UpdateAutoOpsRuleRequest) []command.Command {
 	commands := make([]command.Command, 0)
+
+	// The current web console uses ChangeAutoOpsRuleOpsTypeCommand to update OpsTypes and enable or disable them.
+	// So ActionType is not set and you need to update ActionType according to OpsType.
+	// ToDo: If ChangeAutoOpsRuleOpsTypeCommand is no longer used in the web console,
+	// the following code is no longer needed and should be removed.
+	var opsType *autoopsproto.OpsType = nil
 	if req.ChangeAutoOpsRuleOpsTypeCommand != nil {
 		commands = append(commands, req.ChangeAutoOpsRuleOpsTypeCommand)
+		opsType = &req.ChangeAutoOpsRuleOpsTypeCommand.OpsType
 	}
```

Comment:
How about the following?
```suggestion
	actionType := autoopsproto.ActionType_UNKNOWN
	if req.ChangeAutoOpsRuleOpsTypeCommand != nil {
		commands = append(commands, req.ChangeAutoOpsRuleOpsTypeCommand)
		if req.ChangeAutoOpsRuleOpsTypeCommand.OpsType == autoopsproto.OpsType_DISABLE_FEATURE {
			actionType = autoopsproto.ActionType_DISABLE
		} else if req.ChangeAutoOpsRuleOpsTypeCommand.OpsType == autoopsproto.OpsType_ENABLE_FEATURE {
			actionType = autoopsproto.ActionType_DISABLE
		}
	}
```

Then use `actionType` for ActionType of each commands. We can remove if block in each for loop bellow.